src/skins/vector/css/common.css

```diff
@@ -22,28 +22,23 @@ html {
 }
 
 body {
-    /* Open Sans lacks combining diacritics, so these will fall through
```

Comment:
This comment is very important - it took ages to work out why diacritics were breaking and how to fix them, and it's vital to explain that the font ordering of Arial & Helvetica in the list here is, unintuitively, very important.  Why was it removed, and please can we have it back?